app/(gcforms)/[locale]/(form administration)/form-builder/[id]/edit/components/ElementPanel.tsx

```diff
@@ -103,7 +107,7 @@ export const ElementPanel = ({
         }
       }}
     >
-      <PanelBodyRoot item={item} />
+      <PanelBodyRoot item={item} key={"panel-body-" + item.id} onChangeMade={forceRefresh} />
```

Comment:
I don't think we need the key here do we?

Reply:
Tested, and nope! It was a left over from me testing out the push of data across components. Removed it. :)